crates/swift-integration-tests/src/shared_types/shared_enum.rs

```diff
@@ -22,7 +29,7 @@ mod ffi {
     enum EnumWithNamedData {
         Variant1 { hello: String, data_u8: u8 },
         Variant2 { data_i32: i32 },
-        Variant3,
+        Variant3 { foo: Foo },
```

Comment:
Instead of adding in `Foo` to this, let's create a new enum each time we want to support a new kind of data.

```rust
enum TestEnumWithOpaqueRust {
    Named { data: OpaqueRustForEnumTest },
    Unnamed(OpaqueRustForEnumTest)
}
```

This way people can more easily see that we're explicitly trying to test opaque Rust data.